repose-aggregator/tests/performance-tests/roles/gatling/tasks/install_gatling.yml

```diff
@@ -54,3 +54,11 @@
     dest: /root/gatling/current/conf/gatling.conf
     mode: 0644
     backup: yes
+
+- name: Copy the Repose library for Gatling
```

Comment:
Once again, for this to work, the Jenkins `performance-test` job must be updated to copy the library JAR.

The relative destination for that JAR should be:
```
repose-aggregator/tests/performance-tests/roles/gatling/files
```